### PR TITLE
Added logs upload documentation

### PR DESCRIPTION
**Pull request by domachad**
Added documentation to detail the new logs upload subcommand, which is only available for internal use at Cisco

### CODE REVIEW

File: docs/cli/pyats_logs.rst [hunk 1]
```diff
@@ -87,3 +87,43 @@ The following is an example of what the webpage looks like:
 
 Each tab shows the detailed results of each section of each testscript, as well
 as all commands executed and specific sections of logs.
+
+
+
+pyats logs upload
+---------------
+
+This subcommand will upload a zipped file to Taas Logviewer. It is an internal to
```
Comment (jeaubin) [inline docs/cli/pyats_logs.rst:96]: This command upload a .zip pyATS Archive to TaaS Logviewer.

File: docs/cli/pyats_logs.rst [hunk 2]
```diff
@@ -87,3 +87,43 @@ The following is an example of what the webpage looks like:
...
+
+    Usage:
+      pyats logs upload [archive] [options]
+
+    Description:
+      Upload pyATS generated archive zip files to Taas Logviewer.
+
+      Examples:
+          # uploads an archive zip file to Taas Logviewer (default URL)
+          $ pyats logs upload file.zip
```
Comment (jeaubin) [inline docs/cli/pyats_logs.rst:109]: $ pyats logs upload archive.zip 

Change that for all  instead of just file.zip

**Review by jeaubin — APPROVED** [review]

File: docs/cli/pyats_logs.rst (hunk 1 above)
Comment (domachad) [reply docs/cli/pyats_logs.rst]: fixed

File: docs/cli/pyats_logs.rst (hunk 2 above)
Comment (domachad) [reply docs/cli/pyats_logs.rst]: fixed

File: docs/cli/pyats_logs.rst [hunk 3]
```diff
@@ -87,3 +87,43 @@ The following is an example of what the webpage looks like:
 
 Each tab shows the detailed results of each section of each testscript, as well
 as all commands executed and specific sections of logs.
+
+
+
+pyats logs upload
+---------------
+
+This command will upload a .zip pyATS Archive to TaaS Logviewer. It is an internal to
```
Comment (LukasMcClelland) [inline docs/cli/pyats_logs.rst:97]: This subcommand*

**Review by LukasMcClelland — APPROVED** [review]

**Pull request merged**